### PR TITLE
 toMetadata for String splits long strings

### PR DESCRIPTION
**Pull request by klntsky**
Re-creation of https://github.com/Plutonomicon/cardano-transaction-lib/pull/1344

### Pre-review checklist

- [ ] All code has been formatted using our config (`make format`)
- [ ] Any new API features or modification of existing behavior is covered as defined in our [test plan](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/test-plan.md)
- [ ] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,8 +50,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Changed
 
 ### Removed
+- AnyToMetadataClass and AnyToMetadata type. Neither was really used anywhere
```
Comment (rynoV) [inline CHANGELOG.md:53]: I don't think this is necessary as these weren't in the public interface.

Other than that lgtm, and `make run-ci-actions` is passing locally

Comment (klntsky) [reply CHANGELOG.md:53]: Agreed

File: CHANGELOG.md [hunk 2]
```diff
@@ -50,8 +50,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
...
 
 ### Fixed
+- CIP-25 strings are now being split into chunks whose sizes are less than or equal to 64 to adhere to the CIP-25 standard ([#1343](https://github.com/Plutonomicon/cardano-transaction-lib/issues/1343))
```
Comment (rynoV) [inline CHANGELOG.md:56]: This isn't a general change for CIP-25 strings is it? It's just for the `image` field?

Comment (klntsky) [reply CHANGELOG.md:55]: According to the standard (where necessary). I think this description is good.

Comment (rynoV) [reply CHANGELOG.md:55]: And the `description` field it looks like

Comment (rynoV) [reply CHANGELOG.md:55]: Right makes sense

**Comment by rynoV** [comment]
Rerunning failing CI as the test is passing locally

**Review by rynoV — APPROVED** [review]

**Pull request merged**